include/kf/TreeMap.h

```diff
@@ -101,12 +101,6 @@ namespace kf
             return m_table.deleteElement(Node::fromKey(key));
         }
 
-        bool removeByObject(const V* value)
```

Comment:
We need to return this method back and add a comment about `value` - it must be a value returned previously by other map methods.